include/utils/error/error_manager_req.hpp

```diff
@@ -14,13 +15,12 @@ namespace Everest {
 namespace error {
 
 struct ErrorDatabase;
-struct ErrorTypeMap;
```

Comment:
I don't really get what the idea behind all the forward defines was - I've removed it to just use the ErrorTypeMapPtr type